lib/rules/no-get-by-for-asserting-element-not-present.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const { getDocsUrl } = require('../utils');
+
+const falsyMatchers = ['toBeNull', 'toBeFalsy'];
+
+module.exports = {
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description:
+        'Disallow the use of `getBy*` queries when checking elements are not present',
+      recommended: 'error',
+      url: getDocsUrl('no-get-by-for-asserting-element-not-present'),
+    },
+    messages: {
+      expectQueryBy:
+        'Use `expect(getBy*)` only when elements are present, otherwise use `expect(queryBy*)`.',
```

Comment:
```suggestion
        'Use `getBy*` only when elements are present, otherwise use `queryBy*` when checking that elements are not present.',
```

Reply:
Is `when cheking that elements are not present` necessary? I feel like it's too much, what about: 

`Use getBy* only when checking elements are present, otherwise use queryBy*`